### ISSUE TITLE
two-gene scatterplot

### ISSUE THREAD

**Issue by cgreene** [AlexsLemonade/RNA-Seq-Exercises#13]
Question came up: can you look at the correlation between two target genes. We could put in a gene-gene scatterplot example.

**Comment by cansav09** [comment]
Seurat has a function made for this very thing. I can add in a section in the scRNA-seq stuff.

**Comment by jaclyn-taroni** [comment]
I have a strong preference for using `ggplot2` (or even base graphics) for this as participants that want to make this kind of plot will not necessarily have data appropriate for use with Seurat (e.g., bulk RNA-seq data). There's nothing stopping someone from loading a bulk count matrix into Seurat to make a two-gene scatterplot, but we want to teach generalizable methods wherever possible. We don't need a specialized package to make a scatterplot. I would put this in the afternoon of day 2 where we'll be covering a lot of the ggplot2 stuff.

**Comment by cansav09** [comment]
Yeah, also I tried the `GenePlot` function in Seurat and it is not great. And also as you said, it teaches people nothing that is very generalizable.

**Comment by jaclyn-taroni** [comment]
I've gone in a different (line graph) direction with the ML exercise notebook, so I'm going to remove that from 2019 Chicago.
